server/models/notificationevent.js

```diff
@@ -0,0 +1,33 @@
+export default (sequelize, DataTypes) => {
+  const NotificationEvent = sequelize.define('NotificationEvent', {
+    body: {
+      allowNull: false,
+      type: DataTypes.STRING
+    },
+    url: {
+      allowNull: false,
+      type: DataTypes.STRING
+    },
+    status: {
+      allowNull: false,
+      type: DataTypes.BOOLEAN
+    }
+  }, {});
+  NotificationEvent.associate = (models) => {
+    // associations can be defined here
+    const { User, Notification } = models;
+    NotificationEvent.belongsTo(User, {
+      foreignKey: 'receiverId',
+      onDelet: 'CASCADE'
```

Comment:
it seems you have a typo on line 21 and line 25.
Consider changing this `onDelet` to `onDelete`

Reply:
Thanks for the feedback.